### ISSUE TITLE
Class attribute with `:` is incorrectly detected as a pseudo-class

### ISSUE THREAD

**Issue by AdamWr** [AdguardTeam/FiltersCompiler#219]
It seems that if there is `:` inside class name (I'm not sure if it's only about class name, maybe the same happens with other attributes) then filters compiler consider it as a pseudo-class:
```adblock
! Error: "Unknown pseudo-class ':min-h-' in selector: 'div[class*="md:min-h-[90px] lg:min-h-[250px]"]:has(div[id^="RAD_D_"])'" in the rule: "radio.at,radio.de,radio.dk,radio.es,radio.fr,radio.it,radio.net,radio.pl,radio.pt,radio.se##div[class*="md:min-h-[90px] lg:min-h-[250px]"]:has(div[id^="RAD_D_"])"
radio.at,radio.de,radio.dk,radio.es,radio.fr,radio.it,radio.net,radio.pl,radio.pt,radio.se##div[class*="md:min-h-[90px] lg:min-h-[250px]"]:has(div[id^="RAD_D_"])
```

https://github.com/AdguardTeam/FiltersRegistry/blob/4e5d379cd96e3a26ba40c1fe072d2b871ad25fad/filters/filter_2_Base/diff.txt#L51-L52
